libs/dal/src/repositories/notification-template/notification-template.repository.ts

```diff
@@ -115,7 +149,7 @@ export class NotificationTemplateRepository extends BaseRepository<
     return this.mapEntity(res);
   }
 
-  public static getBlueprintOrganizationId(): string | undefined {
+  public static getBlueprintEnvironmentId(): string | undefined {
```

Comment:
how would I know the `environment id` if I've just forked and deployed it for the first time?

Reply:
I am reverting this change to the organization, but the question will be still valid, and it is good I will add it to the open questions.

Reply:
Dima's [answer](https://www.notion.so/Technical-spec-bd4583dff7f04a63b51001fb3dd684ba?d=a1ae851c7d134e99b35a957fe1be4fa0#28cf65d0f0764628949cb364a6f51f15) ,the newly forked project won't point to the local API it will point to the cloud version.